### PR TITLE
Fix many stylelint items

### PR DESCRIPTION
**Pull request by kendaleiv**

### CODE REVIEW

File: css/scss/_forms.scss [hunk 1]
```diff
@@ -104,17 +107,17 @@ label.select {
...
     @include rotate(90deg);
     right: 0.3em;
-    top: .5em;
-    padding: 0 0 .3em;
+    top: 0.5em;
+    padding: 0 0 0.3em;
     position: absolute;
     pointer-events: none;
   }
-  &:before {
```
Comment (hougasian) [inline css/scss/_forms.scss]: https://css-tricks.com/almanac/selectors/a/after-and-before/#article-header-id-0

`::` vs. `:` if very specific; the standards correct `::` drops support for ie8. 
this is why every pseudo class is specifically `:`
thoughts?

Comment (kendaleiv) [inline css/scss/_forms.scss]: Does any pre-processor we are using happen to handle it? If not, I'm totally fine with relaxing that stylelint rule and changing them back.

Comment (hougasian) [inline css/scss/_forms.scss]: "handle it"?

Comment (kendaleiv) [inline css/scss/_forms.scss]: As in, something modifying the output to support IE8 for those particular things. Assuming we don't have that and don't want to add it (complexity) I'll change them back to single colon and modify the lint rules as necessary.

Comment (hougasian) [inline css/scss/_forms.scss]: No, and no intention of adding additional ie8 anything. Let's relax the rules to accommodate the single colon and note the reasoning.

File: css/scss/_helpers.scss [hunk 2]
```diff
@@ -1,43 +1,94 @@
 // Helper classes
 // Helper classes emulate easily recognizable css
-.left {float: left;}
-.right {float: right;}
-.none {float: none;}
+.left {
```
Comment (hougasian) [inline css/scss/_helpers.scss]: @kendaleiv didn't we discuss _not_ changing in these repetitive helper class instances? It's truly a waste of lines.

File: css/scss/_forms.scss [hunk 3]
```diff
@@ -166,31 +169,28 @@ input[type=range]::-webkit-slider-thumb {
...
-@mixin build-slider($slider-color:null,$slider-color-focus:null) {
-
-}
-
-
-
+// @mixin build-slider($slider-color:null, $slider-color-focus:null) {
+// }
 
 input[type=range]::-moz-range-thumb {
```
Comment (hougasian) [inline css/scss/_forms.scss]: until this can be revisited, L175 down should be removed in _forms.scss.

Comment (kendaleiv) [inline css/scss/_forms.scss]: Can we do that in a different PR?

Comment (hougasian) [inline css/scss/_forms.scss]: certainly!

File: css/scss/_helpers.scss (hunk 2 above)
Comment (kendaleiv) [inline css/scss/_helpers.scss]: Better? I didn't do this in 100% of the possible places, but I can if that's what you'd prefer.

File: css/scss/_cards.scss [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
 // Generate card sizes based on containers
+
 @include responsive-cards(0);
 
 .card {
   background: $card--background-color;
   @include border(right);
   @include border(bottom);
   display: inline-block;
-  float: left;
```
Comment (hougasian) [inline css/scss/_cards.scss]: this breaks cards, replacing
